src/test/nodesessionmetadata_test.cpp

```diff
@@ -28,16 +28,17 @@ using testing::Not;
 using testing::Return;
 
 class NodeSessionMetadataTest : public ::testing::Test {
-    NodeSessionMetadata meta1;
+protected:
+    ExecutionContext context{ExecutionContext::Interface::GRPC, ExecutionContext::Method::Predict};
```

Comment:
Do we need separate context here if we have default:
DEFAULT_CONTEXT?

Reply:
To be added in next metric PR